src/main/data/bai.js

```diff
@@ -40,9 +40,7 @@ function computeIndexChunks(buffer) {
     if (n_intv) {
       var offset = VirtualOffset.fromBlob(view.getBytes(8), 0),
           coffset = offset.coffset + (offset.uoffset ? 65536 : 0);
-      if (coffset) {
-        minBlockIndex = Math.min(coffset, minBlockIndex);
-      }
+      minBlockIndex = coffset ? Math.min(coffset, minBlockIndex) : 65536;
```

Comment:
What's the significance of 65536? (Maybe make a constant here instead of this magic number?)